src/api/spec/factories/kiwi_repository.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :kiwi_repository, class: Kiwi::Repository do
+    association :image, factory: :kiwi_image
+
+    source_path 'http://example.com/'
+    order 1
+    repo_type { Kiwi::Repository::REPO_TYPES.first }
+    replaceable false
+
+    factory :kiwi_repository_with_package do
+      after(:create) do |repository|
+        repository.image.package = create(:package)
```

Comment:
It will be also needed a `repository.reload`. @Ana06 and I decided to leave it as it is.